docs/docs/gatsby-starters.md

```diff
@@ -20,6 +20,14 @@ This downloads the files and initializes the site by running `npm install`
 If you don't specify a custom starter, your site will be created from the
 [default starter](https://github.com/gatsbyjs/gatsby-starter-default).
 
+## Prerequisites
```

Comment:
This is great! Definitely is needed.

I don't think this is the right place for this however — we should have a generic setup Linux for Gatsby pages similar to what we have for Windows https://www.gatsbyjs.org/docs/gatsby-on-windows/

Then this section could link out to those more detailed pages. Probably this section should also hint at the prerequisites e.g. native compilation + Python so people who are running into problems get a hint that they should really go check out the other page in detail for help.

How's that sound?